src/api/app/components/notification_component.rb

```diff
@@ -3,14 +3,15 @@ class NotificationComponent < ApplicationComponent
     'BsRequest' => 'fa-code-pull-request', 'Comment' => 'fa-comments',
     'Package' => 'fa-xmark text-danger',
     'Report' => 'fa-flag', 'Decision' => 'fa-clipboard-check',
-    'Appeal' => 'fa-hand', 'WorkflowRun' => 'fa-book-open'
+    'Appeal' => 'fa-hand', 'WorkflowRun' => 'fa-book-open',
+    'GroupsUser' => 'fa-people-group'
   }.freeze
 
   NOTIFICATION_TITLE = {
     'BsRequest' => 'Request notification', 'Comment' => 'Comment notification',
     'Package' => 'Package notification', 'Report' => 'Report notification',
     'Decision' => 'Report decision', 'Appeal' => 'Decision appeal',
-    'WorkflowRun' => 'Workflow run'
+    'WorkflowRun' => 'Workflow run', 'GroupsUser' => 'Group user notification'
```

Comment:
What is a "Group user"?

Reply:
I changed it to `'GroupsUser' => 'User added/removed on a group'`